lib/pengine/bundle.c

```diff
@@ -2002,3 +1985,20 @@ pe__bundle_active_node(const pe_resource_t *rsc, unsigned int *count_all,
     g_hash_table_destroy(nodes);
     return active;
 }
+
+/*!
+ * \brief Get maximum bundle resource instances per node
```

Comment:
All four of the variant-specific functions can get the `\internal` flag; only the method is public

Reply:
Whoops, copy-and-paste